demisto_sdk/commands/content_graph/parsers/integration_script.py

```diff
@@ -31,9 +32,13 @@ def field_mapping(self):
         super().field_mapping.update({"object_id": "commonfields.id"})
         return super().field_mapping
 
-    @property
-    def docker_image(self) -> str:
-        return get_value(self.yml_data, self.field_mapping.get("docker_image", ""), "")
+    @cached_property
+    def docker_image(self) -> DockerImage:
+        docker_image = (
```

Comment:
What if it is missing?
We have two options:
1. it is a python script which runs on alpine.
2. It is javascript which runs natively.

Maybe it worth distinguish them?

Reply:
what's wrong if the DockerImage will be an empty string and not `None`? how would it make any difference?

Reply:
You are right basically, but this was done because of https://github.com/demisto/demisto-sdk/pull/3958#discussion_r1468853116